client/src/components/screens/PatientVariant/index.jsx

```diff
@@ -65,99 +65,159 @@ const COLUMN_WIDTHS = {
   TINY: 52,
 };
 
-const REPORT_HEADER_NUCLEOTIDIC_VARIATION = 'Variation nucléotidique';
-const REPORT_HEADER_STATUS_PARENTAL_ORIGIN = 'Statut (origine parentale)';
-const REPORT_HEADER_ALLELIC_FREQUENCY = 'Fréquence allélique';
-const REPORT_HEADER_SILICO_PREDICTION = 'Prédiction in silico';
-const REPORT_HEADER_CLIN_VAR = 'ClinVar';
-
 const getValue = curryRight(get)('');
-const valuePresent = x => (x !== undefined && x.trim());
-const join = sep => a => (isArray(a) ? a.join(sep) : '');
+const valuePresent = x => (x !== undefined && x !== '' && x !== null);
```

Comment:
Or (probably being nitpicky about making it a few characters shorter):

```
const valuePresent = x => !_.isNil(x) && x !== ''
```

Reply:
En fait, avec la fonction isNil je n'avais pas le résultat que je voulais. Je vais réessayer pour voir ...

Reply:
```
> const valuePresent = x => !_.isNil(x) && x !== ''
undefined
> valuePresent('')
false
> valuePresent('a')
true
> valuePresent(null)
false
> valuePresent(1)
true
```

Reply:
Pour lodash 4.17.17 sur node 14 anyways

Reply:
Oui, c'est bon, j'ai fait des tests aussi!